src/app/domains/wallets/domains.wallets.enums.ts

```diff
@@ -0,0 +1,5 @@
+export enum NotificationType {
+    Error = 'Error',
+    Warning = 'Warning',
+    Success = 'Success',
+}
```

Comment:
Generally okey, I would only move it to poper directory and change values to lowercase names:
```typescript
export enum NotificationType {
    Error = 'error',
    Warning = 'warning',
    Success = 'success',
}
```